lnbits/db.py

```diff
@@ -173,16 +173,19 @@ async def fetch_page(
         values: Optional[List[str]] = None,
         filters: Optional[Filters] = None,
         model: Optional[Type[TRowModel]] = None,
+        group_by: Optional[str] = None,
     ) -> Page[TRowModel]:
         if not filters:
             filters = Filters()
         clause = filters.where(where)
         parsed_values = filters.values(values)
+        group_by = f"GROUP BY {group_by}" if group_by else ""
```

Comment:
SQL injection risk

Reply:
cant easily be avoided here. this is stuff which orms / query builders would take care of

Reply:
a regex check of this (roughly) form should do it: `^[a-zA-Z_][a-zA-Z0-9_]*$`

Reply:
yeah, we should probably add:

``` python
assert re.fullmatch(r"[a-zA-Z_][a-zA-Z0-9_]*", group_by)
```